pages/platform/kubernetes-k8s/known-limits/guide.en-asia.md

```diff
@@ -47,6 +47,12 @@ Most worker nodes (be them added manually or through cluster autoscaler) are cre
 
 Delivering a fully managed service, including OS and other component updates, you will neither need nor be able to SSH as root into your nodes.
 
+## Data persistence
+
+If an incident is detected by the OVHcloud monitoring, as part of autohealthing, or in case of a version upgrade, the Nodes can be fully reinstalled. 
+
+We advise you to save your data in Persistent Volumes (PV), not to save data directly on Nodes if you don't want to lost your data. Follow our [guide about how to setup and manage Persistent Volumes on OVHcloud Managed Kubernetes](../ovh-kubernetes-persistent-volumes/) for more information.
```

Comment:
"if you don't want to lost your data" should be replaced by "if you don't want to lose your data". Please update it on all versions @scraly :-)

Reply:
I don't see "lose" to fix to "lost" :)

Reply:
That's the other way around, "lost" has to be replaced by "lose".

Reply:
Ok understood and fixed :)